### PR TITLE
[New] Added test cases for record form form subforms subform fields c…

### PR DESCRIPTION
**Pull request by Srinivas9933**
Added test cases for subforms subform fields components violation-title component.

- component.spec.js

### CODE REVIEW

File: app/javascript/components/record-form/form/subforms/subform-fields/components/violation-title/component.spec.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        initialState
+      );
+
+      expect(screen.getByText("This is a title - b123cde")).toBeInTheDocument();
+      expect(screen.getByTestId("violation-title")).toBeInTheDocument();
+    });
+  });
+
+  describe("when violation status and short_id is not present", () => {
+    it("renders a vioaltion title with chip", () => {
```
Comment (dhernandez-quoin) [inline app/javascript/components/record-form/form/subforms/subform-fields/components/violation-title/component.spec.js:80]: This should test that the `Chip` component is not there.

File: app/javascript/components/record-form/form/subforms/subform-fields/components/violation-title/component.spec.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        initialState
+      );
+
+      expect(screen.getByText(/status 1/i)).toBeInTheDocument();
+      expect(screen.getByTestId("violation-title")).toBeInTheDocument();
+    });
+  });
+
+  describe("when violation status is not present", () => {
+    it("renders a vioaltion title with chip", () => {
```
Comment (dhernandez-quoin) [inline app/javascript/components/record-form/form/subforms/subform-fields/components/violation-title/component.spec.js:61]: This should test that the Chip component is there.

**Review by dhernandez-quoin — CHANGES_REQUESTED** [review]

File: app/javascript/components/record-form/form/subforms/subform-fields/components/violation-title/component.spec.js (hunk 1 above)
Comment (Srinivas9933) [reply app/javascript/components/record-form/form/subforms/subform-fields/components/violation-title/component.spec.js:77]: Done

File: app/javascript/components/record-form/form/subforms/subform-fields/components/violation-title/component.spec.js (hunk 2 above)
Comment (Srinivas9933) [reply app/javascript/components/record-form/form/subforms/subform-fields/components/violation-title/component.spec.js]: Done

**Pull request merged**